examples/CSharp/Tutorial 2.2 - Camera/Program.cs

```diff
@@ -28,7 +28,7 @@ class Program
         //Setup the cameras location, directions, and movement speed
         private static Vector3 CameraPosition = new Vector3(0.0f, 0.0f, 3.0f);
         private static Vector3 CameraFront = new Vector3(0.0f, 0.0f, -1.0f);
-        private static Vector3 CameraUp = new Vector3(0.0f, 1.0f, 3.0f);
+        private static Vector3 CameraUp = new Vector3(0.0f, 1.0f, 0.0f);
```

Comment:
Is there no UnitY constant?

Reply:
There is, I hadn't used it to keep it standard with the other two, but I can change this

Reply:
Overall I'd say use the UnitX/Y/Z constants as much as possible, doing `-` or `3 * UnitX` > using the ctors

Reply:
That is the approach all of the later tutorials take, this one was done specifically to match the contents of the LearnOpenGL tutorial that first introduces cameras so that whoever was following along could see the similarities line by line